bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -0,0 +1,232 @@
+			setButtonClickListener("mediumBtn", mediumBtn, brushDialog, true);
+
+			ImageButton largeBtn = (ImageButton) brushDialog.findViewById(R.id.large_brush);
+			setButtonClickListener("largeBtn",largeBtn,brushDialog, true);
+
+			brushDialog.show();
+
+		} else if (view.getId() == R.id.erase_btn){
+
+			ImageButton smallBtn = (ImageButton) brushDialog.findViewById(R.id.small_brush);
+			setButtonClickListener("smallBtn", smallBtn, brushDialog, false);
+
+			ImageButton mediumBtn = (ImageButton) brushDialog.findViewById(R.id.medium_brush);
+			setButtonClickListener("mediumBtn", mediumBtn, brushDialog, false);
+
+			ImageButton largeBtn = (ImageButton) brushDialog.findViewById(R.id.large_brush);
+			setButtonClickListener("largeBtn",largeBtn,brushDialog, false);
+
+			brushDialog.show();
+		}
+
+		else if (view.getId()==R.id.sendEmail){
+			saveSendScreenshotAndLog();
+		}
+
+	}
+
+	public void paintClicked(View view){
+		drawView.setBrushSize(drawView.getLastBrushSize());
+
+		if(view!=currPaint){
+			ImageButton imgView = (ImageButton)view;
+			String clickedButtonColor = view.getTag().toString();
+			drawView.setColor(clickedButtonColor);
+			imgView.setImageDrawable(getResources().getDrawable(R.drawable.paint_pressed));
+			currPaint.setImageDrawable(getResources().getDrawable(R.drawable.paint));
+			currPaint=(ImageButton)view;
+		}
+	 }
+
+	private Uri getImageUri(Context inContext, Bitmap inImage) {
+		ByteArrayOutputStream bytes = new ByteArrayOutputStream();
+		inImage.compress(Bitmap.CompressFormat.JPEG, 100, bytes);
+		String path = MediaStore.Images.Media.insertImage(inContext.getContentResolver(), inImage, "Title", null);
+		return Uri.parse(path);
+	}
+
+	private void saveSendScreenshotAndLog() {
+		AlertDialog.Builder sendDialog = new AlertDialog.Builder(this);
+
+		sendDialog.setTitle(getString(R.string.send_annotated_screenshot));
+		sendDialog.setMessage(getString(R.string.attach_annotated_screenshot_to_email));
+		sendDialog.setPositiveButton(getString(R.string.yes), new DialogInterface.OnClickListener(){
+			public void onClick(DialogInterface dialog, int which){
+				drawView.setDrawingCacheEnabled(true);
+
+				String imgSaved = MediaStore.Images.Media.insertImage(
+					getContentResolver(), drawView.getDrawingCache(),
+					ScreenshotUtil.IMAGE_FILE_NAME, ScreenshotUtil.IMAGE_DESCRIPTION);
+				Bitmap screenshotBitmap = drawView.getDrawingCache();
+
+				Uri bitmapUri = getImageUri(getApplicationContext(), screenshotBitmap);
+
+				if (imgSaved != null) {
+					Toast savedToast = Toast.makeText(getApplicationContext(),
+						"Drawing saved to Gallery!", Toast.LENGTH_SHORT);
+
+
+					File log = LogcatUtil.saveLogcatToFile(getApplicationContext());
+					Uri logUri = Uri.fromFile(log);
+					SendEmailUtil.sendEmailWithScreenshot(getBaseContext(), bitmapUri, logUri,
+						FeedbackEmailFlowManager.emailAddresses, FeedbackEmailFlowManager.emailSubjectLine);
```

Comment:
We can get rid of using the FeedbackEmailFlowManager now. We can create this constant somewhere else.